spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/SparkTableUtil.java

```diff
@@ -405,14 +405,16 @@ private static Iterator<ManifestFile> buildManifest(
    * @param partitionFilter only import partitions whose values match those in the map, can be
    *     partially defined
    * @param checkDuplicateFiles if true, throw exception if import results in a duplicate data file
+   * @param parallelism Controls max concurrency of file reads per partition
    */
   public static void importSparkTable(
```

Comment:
Unfortunately this is a public api, We should probably add a new signature and deprecate the old one, or keep the old one. I think in this case keeping the old one without deprecation is fine but we probably should not break binary compatibility unless we have to.

Reply:
Thanks again for your reviews, this PR's changes is almost same with the older #3973 :)